ui_tests/pages/profiles_page.py

```diff
@@ -0,0 +1,299 @@
+import time
+
+import pages.helper as helper
+from logger.logger import Logger
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from webdriver.webriver_factory import WebDriverFactory
+
+logger = Logger().get_logger()
+driver = WebDriverFactory.get_driver()
+
+
+class ProfilesPage:
+    def checkIfProfilesTableIsDisplayed(self):
+        logger.info("Check if groups page is displayed")
+        profiles_table_xpath = "//div[@data-test='sc4snmp:profiles-table']"
+        profiles_container = driver.find_element(By.XPATH, profiles_table_xpath)
+        return profiles_container.is_displayed()
+
+    def click_add_profile_button(self):
+        logger.info("Click Add New Profile button")
+        xpath = "//button[@data-test='sc4snmp:new-item-button']"
+        btn = driver.find_element(By.XPATH, xpath)
+        btn.click()
+
+    def click_submit_button(self):
+        logger.info("Click Submit button")
+        xpath = "//button[@data-test='sc4snmp:form:submit-form-button']"
+        btn = driver.find_element(By.XPATH, xpath)
+        btn.click()
+        time.sleep(5)  # wait for profile to be shown on the list
```

Comment:
Similar comment about `time.sleep` in this file

Reply:
there is no loading element, and method used in multiple places 
hard to add explicit wait here